src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -101,6 +101,11 @@ private static void InitSentryAndroidSdk(SentryOptions options)
             {
                 o.BeforeSend = new BeforeSendCallback(beforeSend, options, o);
             }
+            else
+            {
+                // Prevent JavaBackgroundThread crashes from creating duplicated event: one UnhandledException one AppDomain.UnhandledException
+                o.BeforeSend = new BeforeSendCallback(((_, _) => null), options, o);
```

Comment:
Doesn't this just tell the Android SDK to discard all exceptions? Are there no circumstances in which we want the Android SDK to catch exceptions?

Reply:
> Are there no circumstances in which we want the Android SDK to catch exceptions?

In this case, they can set an `options.Native.EnableBeforeSend` to true. But I can't think of  any use case in which they would since the event thrown by dotnet has a more detailed set of info

Reply:
But what about java exceptions that don't come from a `JavaBackgroundThread`? This would discard those too right? It's just discarding all and any events that get captured by the Java SDK.

And if the user has set any custom BeforeSend callback, this workaround wouldn't work right???

Reply:
All Java event throws a dotnet event, as far as I can tell. 

> if the user has set any custom BeforeSend callback, this workaround wouldn't work right???

it would work as long as `Native.EnableBeforeSend` is not set to true. if it's set to true, it means they technically want the Java event to appear

Reply:
> All Java exceptions event throws a dotnet event, as far as I can tell.

I think this wouldn't:
```
try 
{
    // Do something crazy
}
catch(Exception e)
{
    SentrySdk.CaptureException(e);
}
```

But I think this would:
```
try 
{
    // Do something crazy
}
catch(Exception e)
{
    SentrySdk.CaptureException(e);
    throw e;
}
```


So we can't assume that all java exceptions will result in a corresponding .NET exception and we can't just throw away all the Java exceptions.

Reply:
> All Java exceptions event throws a dotnet event, as far as I can tell.

If I write in Java/Kotlin `Sentry.captureException` does that go through the C# BeforeSend? I'd be surprised.

A NDK crash, does it go through the C# BeforeSend? Also would surprise me.